src/protocol.py

```diff
@@ -9,20 +9,33 @@
 import random
 import socket
 import sys
+import six
 import time
 from binascii import hexlify
 from struct import Struct, pack, unpack
 
-import defaults
-import highlevelcrypto
-import state
-from addresses import (
-    encodeVarint, decodeVarint, decodeAddress, varintDecodeError)
-from bmconfigparser import BMConfigParser
-from debug import logger
-from fallback import RIPEMD160Hash
-from helper_sql import sqlExecute
-from version import softwareVersion
+try:
+    import defaults
+    import highlevelcrypto
+    import state
+    from addresses import (
+        encodeVarint, decodeVarint, decodeAddress, varintDecodeError)
+    from bmconfigparser import BMConfigParser
+    from debug import logger
+    from fallback import RIPEMD160Hash
+    from helper_sql import sqlExecute
+    from version import softwareVersion
+except ImportError:
+    from . import defaults
+    from . import highlevelcrypto
+    from . import state
```

Comment:
These imports merge:
```python
from . import defaults, highlevelcrypto, state
```

Reply:
That leads to Code quality error: Pylint E401 multiple imports on one line

Reply:
No, it's the same as ```from addresses import ...``` above.

Reply:
Yes, because it's from the same module ```addresses``` but when importing different modules keeping them separate is best practice as per https://www.python.org/dev/peps/pep-0008/#imports

Also, I tried what you mentioned I got ```Pylint E401 multiple imports on one line```

Reply:
`.` is an abbrev to `pybitmessage` (i.e. this package) in this case. But in any case, you don't need theses `try: import .. except:` at all.

Reply:
Yes, understood. Saw your comment on #1792. I'm making those changes.

Reply:
The proper change is to revert this all together. Did you try to rebase?

Reply:
Yes

Reply:
... and it's still failing, and my #1788 passes with no pasta